src/schema/resolvers/user/mutation.resolver.js

```diff
@@ -0,0 +1,39 @@
+import { ApolloError } from "apollo-server-errors";
+import { hash } from "utils/auth";
+
+const updateUser = async (__, args, context) => {
+  const password = await hash(args.password);
+
+  const { data: result } = await context.database.users.update(context.userId, {
```

Comment:
I think you need to check if context.userId exists first (it can be null if authorization header isn't defined)